modules/configuration/js/configuration_helper.js

```diff
@@ -81,13 +81,12 @@ $(function () {
             data: form,
             success: function () {
                 var html = "<label>Submitted</label>";
-                $(html).hide().appendTo('.submit-area').fadeIn(500).delay(1000).fadeOut(500)
-                $('input[type="reset"]').attr('disabled','disabled');
+                $(html).hide().appendTo('.submit-area').fadeIn(500).delay(1000).fadeOut(500);
+                location.reload();
```

Comment:
This shouldn't reload the page, the data is maintained by ajax.

Reply:
when user click the remove (it will call this ajax) or add instruments, the user also changes the other data at the same time, it will break the reset button. That is why I put reload function.

Reply:
That sounds like it's just bandaging over a much deeper problem, but okay, for the interest of getting this release done that's fine